src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(x) {
+    return (y) => {
+      count++;
+
+      if (count <= 3) {
+        return x + y;
+      }
+
+      if (count > 3 && count % 2 === 0) {
```

Comment:
you don't have to check twice if the count is greater than 3

Reply:
Good point